spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -401,6 +401,20 @@ public void validate(InjectionPoint injectionPoint);
 
 ==== Firing an event
 
+The method `BeanManager.getEvent()` returns an instance of `Event` with specified type `java.lang.Object` and specified qualifier `@Default`.
+
+[source, java]
+----
+Event<Object> getEvent();
+----
+
+The returned instance can be used like a standard `Event` as described in <<events>>.
+
+===== `fireEvent()` method
```

Comment:
Merged too fast. In fact I propose a more straightforward deprecation text in PR #351